README.md

```diff
@@ -1,2 +1,110 @@
-# terraform-google-log-export
-A Cloud Foundation Toolkit Module: Opinionated Google Cloud Platform project creation and configuration with Shared VPC, IAM, APIs, etc.
+# Terraform Logsink Module
+
+## Requirements
+### Terraform plugins
+- [Terraform](https://www.terraform.io/downloads.html) 0.10.x
+- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v1.8.0
+
+### Configure a Service Account
+In order to execute this module you must have a Service Account with the following:
+
+#### Roles
+The service account with the following roles:
+- roles/logging.configWriter on the logsink project (to create logsinks)
+- roles/serviceAccount.admin on the logsink project (to create a service account for the log sinks)

```

Comment:
Make it clear this is only required if you want to automatically create a subscriber service account for pubsub.

Reply:
Those 2 roles are actually required for all sinks.

Reply:
Precisions:
- project-level sink: grant those roles on the logsink project
- folder-level sink: grant those roles on the folder-level
- org-level sink: grant those roles on the org-level